### DISCUSSION TITLE
Join tables across different schemas

### DISCUSSION THREAD

**Discussion by sindreolsenhk** [PostgREST/postgrest#2769] [Q&A] [answered]
I was wondering if there was a way to join two tables beloning to two different schemas in one single request. 
So far I have not been able to do this by having all the required schemas in the _db-schemas_ variable in the config file (or via environment variable). Views that are "pre-joined" could work but would result in a lot of views and is not wanted.
From the documentation it seems you have to switch schemas per request, but maybe there is a workaround.

**Comment by steve-chavez** [comment, marked as answer]
It's not possible to do it directly right now, see https://github.com/PostgREST/postgrest/issues/2346

> From the documentation it seems you have to switch schemas per request, but maybe there is a workaround.

If you have a `schema1.table1` related(with foreign key) to `schema2.table2`, then you could create a view of `schema1.table1` into `schema2`. PostgREST will discover the tables' relationship and you'll be able to do resource embedding on them.

See https://postgrest.org/en/v8.0/how-tos/embedding-table-from-another-schema.html

**Comment by skamensky** [comment]
Would still love to see this feature!
